src/components/atoms/404/NotFound.stories.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import NotFound from './NotFound';
```

Comment:
It's fantastic that we have storybook in the project 🚀🚀🚀 We should definitely add info about Storybook to readme, best with information how to run it 💪